src/app/services/calendar/calendar.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { environment } from '../../../environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CalendarService {
+
+  constructor(private http: HttpClient) {}
+
+  getEvents() {
```

Comment:
Should we just merge this into `EventsService`? We can rename this to `getGoogleCalendarEvents()` and park it inside `EventsService`. Also since we have an `Event` type, we also need to transform this into `Promise<Event[]>`. Ideally we need to combine both entries from Eventbrite and Google Calendar.

Reply:
Resolved